G1ANT.Addon.Images/G1ANT.Addon.Images.Tests/Commands/image.expected/ShouldFindImage_WhenImageIsExpected.G1ANT

```diff
@@ -0,0 +1,11 @@
+﻿addon images version 2.205.17304.1000
+
+♥guid = ⊂Guid.NewGuid().ToString()⊃
+♥imageFile = ♥environment⟦temp⟧\♥guid.jpg
+file.download https://jeremykun.files.wordpress.com/2012/01/img49.png filename ♥imageFile
+program mspaint.exe arguments ♥imageFile
+delay milliseconds 300
```

Comment:
Are these delays really necessary?

Reply:
after opening mspaint or other programs windows is resizing them

Reply:
I have try to use waitfor.ui ‴/ui[@class='MSPaintApp']‴ instead of delay, its not working